plugins/woocommerce/src/Admin/Features/Blueprint/Init.php

```diff
@@ -116,10 +116,41 @@ public function add_woo_exporters( array $exporters ) {
 	 * @return array
 	 */
 	public function get_step_groups_for_js() {
-		$all_plugins    = $this->wp_get_plugins();
-		$active_plugins = array_intersect_key( $all_plugins, array_flip( get_option( 'active_plugins', array() ) ) );
+		$plugins        = $this->wp_get_plugins();
+		$themes         = $this->wp_get_themes();
+		$active_plugins = get_option( 'active_plugins', array() );
 		$active_theme   = $this->wp_get_theme();
 
+		$plugins = array_map(
+			function ( $key, $plugin ) use ( $active_plugins ) {
+				return array(
+					'id'      => $key,
+					'label'   => $plugin['Name'],
+					'checked' => in_array( $key, $active_plugins, true ),
+				);
+			},
+			array_keys( $plugins ),
+			$plugins
+		);
+
+		usort(
+			$plugins,
+			function ( $a, $b ) {
+				return $b['checked'] <=> $a['checked'];
+			}
+		);
+
+		$themes = array_map(
+			function ( $theme ) use ( $active_theme ) {
+				return array(
+					'id'      => $theme->get_stylesheet(),
+					'label'   => $theme->get( 'Name' ),
+					'checked' => $theme->get_stylesheet() === $active_theme->get_stylesheet(),
+				);
+			},
+			$themes
+		);
+
```

Comment:
For consistency, what do you think of sorting `$themes` here similar to `$plugins`